src/sass/partials/components/_nav.scss

```diff
@@ -41,3 +41,48 @@
     }
   }
 }
+
+.breadcrumb {
+
+  &--list {
+    background-color: transparent;
+    display: flex;
+    margin-top: 0;
+    transform-origin: top center;
```

Comment:
You're not using any CSS transforms in here; why did you set the transform origin?